### ISSUE TITLE
Dropout Layer in CNN

### ISSUE THREAD

**Issue by zfang** [ChenRocks/fast_abs_rl#3]
Hello,

Here in the CNN model, it seems the dropout layer is placed before the convolutional layer as opposed to after the max pooling layer as per Kim 2014: https://github.com/ChenRocks/fast_abs_rl/blob/b1b66c180a135905574fdb9e80a6da2cabf7f15c/model/extract.py#L28

Is this intentional or a bug? If it's intentional, what is the rationale behind it?

Thanks,

Felix

**Comment by ChenRocks** [comment]
I tried applying dropout on the word embeddings hoping that model will not overfit to a particular word. In the end it turns out dropout did not help the final performance though. Our best models are trained without dropout so this does not make any difference nor a bug.
Feel free to fork and make changes to see if adding dropout to the CNN output improves the model :)

**Comment by ChenRocks** [comment]
closing this issue now. If you find out changing the dropout layer helps, feel free to reply and I'll open the issue again for discussion.
